testing-common/src/main/groovy/io/opentelemetry/instrumentation/test/asserts/TraceAssert.groovy

```diff
@@ -28,8 +27,8 @@ class TraceAssert {
                           @ClosureParams(value = SimpleType, options = ['io.opentelemetry.instrumentation.test.asserts.TraceAssert'])
                           @DelegatesTo(value = TraceAssert, strategy = Closure.DELEGATE_FIRST) Closure spec) {
     def spans = getTrace(spanSupplier, traceId)
-    Stopwatch stopwatch = Stopwatch.createStarted()
-    while (spans.size() < expectedSize && stopwatch.elapsed(TimeUnit.SECONDS) < 10) {
+    def startTime = System.currentTimeMillis()
```

Comment:
Use nanoTime for measurements, not wall time (this would be the previous Guava behavior)

Same for all the similar code

Reply:
👍 

Done!